pkg/router/template/configmanager/haproxy/manager.go

```diff
@@ -1105,6 +1111,15 @@ func modAnnotationsList(termination routev1.TLSTerminationType) []string {
 		"router.openshift.io/haproxy.health.check.interval",
 	}
 
+	// allow adding list of extra annotations for HAProxy blueprint
+	listExtraAnnotations := os.Getenv("ROUTER_BLUEPRINT_CUSTOM_ANNOTATIONS")
+	for _, entry := range strings.Split(listExtraAnnotations, ",") {
+		if v := strings.Trim(entry, " "); len(v) > 0 {
+			glog.V(4).Infof("ROUTER_BLUEPRINT_CUSTOM_ANNOTATIONS has anotation %s", entry)
```

Comment:
Maybe `glog.V(6)`? Otherwise this is going to generate a lot of log entries if the env variable is set.

Reply:
Yes, you are totally right, I just fixed it, thanks for your comments :)